bin/create-action.js

```diff
@@ -0,0 +1,164 @@
+ *
+ * @returns {Promise<PromptAnswers>} An object containing prompt answers.
+ */
+const getActionMetadata = async () => {
+  try {
+    return await prompt([
+      {
+        type: 'input',
+        name: 'name',
+        message: 'What is the name of your action?',
+        initial: 'Your action name',
+        validate: isNotEmpty
+      },
+      {
+        type: 'input',
+        name: 'description',
+        message: 'What is a short description of your action?',
+        initial: 'A description of your action',
+        validate: isNotEmpty
+      },
+      {
+        type: 'autocomplete',
+        name: 'icon',
+        message: 'Choose an icon for your action. Visit https://feathericons.com for a visual reference.',
+        choices: icons,
+        limit: 10
+      },
+      {
+        type: 'autocomplete',
+        name: 'color',
+        message: 'Choose a background color background color used in the visual workflow editor for your action.',
+        choices: colors
+      }
+    ])
+  } catch (err) {
+    // When prompt() rejects, that means the user has pressed ctrl+c to cancel input.
+    throw new Error('Cancelled. Maybe next time!')
+  }
+}
+
+/**
+ * Creates a Dockerfile contents string, replacing variables in the Dockerfile template
+ * with values passed in by the user from the CLI prompt.
+ *
+ * @param {PromptAnswers} answers The CLI prompt answers.
+ * @returns {Promise<string>} The Dockerfile contents.
+ */
+const createDockerfile = async (answers) => {
+  const dockerfileTemplate = await readTemplate('Dockerfile')
+  return dockerfileTemplate
+    .replace(':NAME', answers.name)
+    .replace(':DESCRIPTION', answers.description)
+    .replace(':ICON', answers.icon)
+    .replace(':COLOR', answers.color)
+}
+
+/**
+ * Creates a `package.json` object with the latest version
+ * of `actions-toolkit` ready to be installed.
+ *
+ * @param {string} name The action package name.
+ * @returns {object} The `package.json` contents.
+ */
+const createPackageJson = (name) => {
+  const { version } = require('../package.json')
+  return {
+    name,
+    private: true,
+    main: 'index.js',
+    dependencies: {
+      'actions-toolkit': `^${version}`
+    }
+  }
+}
+
+/**
+ * Runs the create action CLI prompt and bootstraps a new directory for the user.
+ *
+ * @public
+ * @param {string[]} argv The command line arguments to parse.
+ * @returns {Promise<void>} Nothing.
+ */
+const createAction = async (argv) => {
+  const args = minimist(argv)
+  const directoryName = args._[0]
+  if (!directoryName || args.help) {
+    console.log(`\nUsage: npx actions-toolkit <name>`)
+    return process.exit(1)
+  }
+  const base = path.join(process.cwd(), directoryName)
+  try {
+    console.log(`Creating folder ${base}...`)
+    await mkdir(base)
+  } catch (err) {
+    if (err.code === 'EEXIST') {
+      throw new Error(`Folder ${base} already exists!`)
+    } else {
```

Comment:
Codecov is failing because this else block does not have a test. Will push up a test to cover this.